cli/src/lib/storage/storage-props.ts

```diff
@@ -11,6 +11,36 @@ import { ArrayType } from "./types/array-type";
 import { DIAMOND_STORAGE_SLOT } from "./storage-changes";
```

Comment:
All the changes in this file are moving specific contract fields to this constant, in order to reference them outside.

Reply:
How is this meant to be maintained over different releases?

Reply:
This is a good point.

They say in the source code of the contracts that the main diamond storage is going to keep "append only" for ever. Like, they should never change the layour, just append to the end. 

This makes this pretty stable, but still, there is a problem, when they add new things we don't know about them. My idea for that was creating this array dynamically, using one of the artifacts generated by the compiler that has the layout of every struct.

Reply:
Sounds like their deploy process will require manual QA and dev for each release then to keep this tool compatible (potentially)